watertap/unit_models/pressure_exchanger.py

```diff
@@ -201,24 +232,46 @@ def build(self):
 
         units_meta = self.config.property_package.get_metadata().get_derived_units
 
-        self.efficiency_pressure_exchanger = Var(
-            self.flowsheet().config.time,
-            initialize=0.95,
-            bounds=(1e-6, 1),
-            domain=NonNegativeReals,
-            units=pyunits.dimensionless,
-            doc="Pressure exchanger efficiency",
-        )
+        if self.config.pressure_exchange_calculation is PressureExchangeType.efficiency:
+            self.efficiency_pressure_exchanger = Var(
+                self.flowsheet().config.time,
+                initialize=0.95,
+                bounds=(1e-6, 1),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="Pressure exchanger efficiency",
+            )
+        elif (
+            self.config.pressure_exchange_calculation
+            is PressureExchangeType.high_pressure_difference
+        ):
+            self.high_pressure_difference = Var(
+                self.flowsheet().config.time,
+                initialize=0.8e5,
+                bounds=(1e-6, None),
+                domain=NonNegativeReals,
+                units=pyunits.Pa,
+                doc="High pressure difference",
```

Comment:
Remind me- is this meant to be the pressure drop on the high-pressure side of the ERD? Assuming that's the case, why not use `deltaP` which should exist on the `ControlVolume `representing the high-pressure side?

Reply:
OK- seems like this is not just pressure drop on the high-pressure control volume.